### DISCUSSION TITLE
show workspace id too, instead of just workspace name

### DISCUSSION THREAD

**Discussion by erlangparasu** [Kong/insomnia#3562] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when workspace have same name.

**Describe the solution you'd like**
A clear diference between workspaces names

**Describe alternatives you've considered**
Show workspace id + name
eg:
`-----------------------`
`wrk_qwer MyWorkspaceOne`
`-----------------------`
`wrk_asdf MyWorkspaceOne`
`-----------------------`
`wrk_zxcv MyWorkspaceOne`
`-----------------------`

**Comment by erlangparasu** [comment]
or just give different color indicator for other workspace that have the same name

**Comment by erlangparasu** [comment]
[screenshot](https://imgur.com/E2DYzgk)

**Reply by erlangparasu** [reply]
must i click one by one box with the same name?

**Comment by dimitropoulos** [comment]
I'm curious: could you please tell me a little more about the deeper use-case.  Why do you have multiple workspaces with the same name?

**Reply by erlangparasu** [reply]
> I'm curious: could you please tell me a little more about the deeper use-case. Why do you have multiple workspaces with the same name?

when logging in using insomnia account, some workspaces sync automatically. and displays multiple workspace with same names in the list.

**Reply by dimitropoulos** [reply]
The intended primary mechanism for telling workspaces apart from each other is to use the renaming functionality.  You can access this by the three dots in the upper right corner of the card in the dashboard.  Does this solve your use-case's problem?
